src/components/Map/Map.js

```diff
@@ -40,20 +40,29 @@ const Map = React.forwardRef((props, ref) => {
                 }
                 map.fitBounds(bounds);
             });
+        } else if (defaultArea.address) {
+            const { Geocoder } = window.google.maps;
+            const geocoder = new Geocoder();
+
+            geocoder.geocode({ address: defaultArea.address }, (result, status) => {
+                if (status === 'OK') {
+                    map.fitBounds(result[0].geometry.viewport);
```

Comment:
is there any chance of either `result[0]` or `geometry` being falsy values or it's 100% safe to do that without any additional checking?

Reply:
if status is 'OK' then as far as I understand the docs, it should be ok